### DISCUSSION TITLE
Timeframe for an updated version of Plaster

### DISCUSSION THREAD

**Discussion by tstager** [PowerShellOrg/Plaster#413] [General]
I was just wondering if you guys had an idea for when you plan to release an update (just a very general idea of a timeframe). I am extremely excited to see that this is back under active development.

**Comment by tstager** [comment]
Any update on the status?

**Comment by tstager** [comment]
Checking back a few years later. I see there has been some work on a version 2. It looks good. Any idea when a general release is going to happen?

**Reply by HeyItsGilbert** [reply]
v2.0.0-beta1 was just released this morning. Feedback is welcome!

**Reply by tstager** [reply]
ok cool. I will check it out.
